packages/provider/src/app/utils.ts

```diff
@@ -73,14 +73,15 @@ export const allowedTransactionKeys: { [key: string]: boolean } = {
   nonce: true,
   to: true,
   value: true,
+  from: true,
 }
 
 export function serializeEthSignTransaction(transaction): Bytes {
   const nonce = zeroPad(transaction.nonce, 32)
   const gasLimit = zeroPad(transaction.gasLimit, 32)
   const gasPrice = zeroPad(transaction.gasPrice, 32)
   const chainId = zeroPad(transaction.chainId, 32)
-  const to = hexStrToBuf(transaction.to)
+  const to = hexStrToBuf(transaction.to) // this is empty for contract creation
```

Comment:
This is ABI encoding, if `to` is `null`, should it commit to `0x00..00` or skip including `to` in the signature hash entirely?